public/js/components/notification.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+import { gettext } from 'utils';
+
+import cx from 'classnames';
+
+
+export default class Notification extends Component {
+
+  static propTypes = {
+    autoHide: PropTypes.bool,
+    handleAutoHide: React.PropTypes.func,
+    handleDismissClick: React.PropTypes.func,
+    text: PropTypes.string.isRequired,
+    type: React.PropTypes.oneOf(['info', 'error']),
+    userDismissable: React.PropTypes.bool,
+  }
+
+  static defaultProps = {
+    autoHide: true,
+    type: 'info',
+    userDismissable: false,
+  }
+
+  componentDidMount() {
+    if (this.props.userDismissable === false &&
+        this.props.autoHide === true) {
+      this.props.handleAutoHide({delay: 5000});
+    }
```

Comment:
actually, I'll probably want a non-dimissable notice that doesn't auto hide for https://github.com/mozilla/payments-ui/issues/411